src/js/view/components/LogViewerList.js

```diff
@@ -82,7 +80,11 @@ const LogViewerList = props => {
         ref={listRef}
         items={props.lines}
         onRenderCell={_onRenderCell}
-        style={{ display: 'inline-block', minWidth: '100%' }}
+        style={{
+          display: 'inline-block',
+          minWidth: '100%'
+        }}
+        version={props.markedHighlight}
```

Comment:
Should this be here?

Reply:
Huh, that's weird, I wonder how it got there, pretty sure I stashed my changes before changing branch. I'll fix it now.

Reply:
Very good catch!!

Reply:
Git works in mysterious ways